### PR TITLE
Fix java.lang.ClassCastException: java.lang.Long cannot be cast to ja…

### PR DESCRIPTION
**Pull request by roesslerj**
…va.lang.Double

	at de.retest.web.WebElementPeer.toInt(WebElementPeer.java:116) [classes/:na]

### CODE REVIEW

File: src/main/java/de/retest/web/WebElementPeer.java [hunk 1]
```diff
@@ -113,7 +113,7 @@ private int toInt( final Object value ) {
 			return Math.toIntExact( Math.round( (Double) value ) );
 		}
 		if ( value instanceof Long ) {
-			return Math.toIntExact( Math.round( (Double) value ) );
+			return Math.toIntExact( Math.round( (Long) value ) );
```
Comment (beatngu13) [inline src/main/java/de/retest/web/WebElementPeer.java]: What I don't get: The method `toInt( Object )` is only invoked by `retrieveOutline()`. Here, the method parameter of `toInt( Object )` is always taken from the map `webData`, which is of type `Map<String, String>`. Hence, the parameter is always a string, which is why only the 2nd `if` statement (`value instanceof String`) will be triggered?

Comment (roesslerj) [reply src/main/java/de/retest/web/WebElementPeer.java]: I can't tell you exactly in which situation that occurred, but that error was actually triggered...

Comment (roesslerj) [reply src/main/java/de/retest/web/WebElementPeer.java]: So this is no "theoretical fix", but an actual fix to an actual error that occurred in the field...

Comment (roesslerj) [reply src/main/java/de/retest/web/WebElementPeer.java]: Can we have any other changes as a separate PR and now just accept that fix and deploy it?

Comment (beatngu13) [reply src/main/java/de/retest/web/WebElementPeer.java]: I see … will merge, but can you show the actual error message and/or how/where the error was?

**Pull request merged**